### PR TITLE
Policies: Restructure and add docs on how to specify version support post-36

### PR DESCRIPTION
**Pull request by rdimaio**

### CODE REVIEW

File: docs/operator/policy_packages.md [hunk 1]
```diff
@@ -33,43 +33,67 @@ is suffixed with the VO name (for example, `package-vo1` or
 The structure of a policy package is very simple. It can contain the
 following:
 
+- An `__init__.py` file that:
+  - indicates the supported Rucio version via the `SUPPORTED_VERSION` field;
+  - indicates the algorithms provided by the package (optional)
 - A `permission.py` module implementing permission
   customisations (optional).
 - A `schema.py` module implementing schema customisations (optional).
```
Comment (voetberg) [inline docs/operator/policy_packages.md:41]:
```suggestion
- A `schema.py` module implementing schema customization (optional).
```

File: docs/operator/policy_packages.md [hunk 2]
```diff
@@ -109,32 +133,28 @@ In all cases the names used to register the functions (e.g. `voname_extract_scop
...
-be uploaded directly to non-deterministic storage,
-`non_deterministic_pfn` algorithms are only ever called for
-replications, but `lfn2pfn` algorithms can also be called for
-initial uploads.
+The `permission.py` and `schema.py` modules are optional; an experiment
+that does not need to customise these modules can omit one or both of
+them from the policy package and the Rucio generic versions will be
+used instead. If these modules are required, the easiest way to create
+them is to modify the generic versions from the Rucio codebase. These
+can be found in `lib/rucio/core/permission/generic.py` and
```
Comment (voetberg) [inline docs/operator/policy_packages.md:143]: Is it worth adding a permalink to them?

File: docs/operator/policy_packages.md [hunk 3]
```diff
@@ -33,43 +33,67 @@ is suffixed with the VO name (for example, `package-vo1` or
 The structure of a policy package is very simple. It can contain the
 following:
 
+- An `__init__.py` file that:
```
[the remark this answers is not in the document]
Comment (rdimaio) [reply docs/operator/policy_packages.md:36]: Could you clarify? This `__init__.py` is in a policy package, not in the Rucio server, so it'd be in a separate module

File: docs/operator/policy_packages.md (hunk 2 above)
Comment (rdimaio) [reply docs/operator/policy_packages.md]: Added in latest revision

File: docs/operator/policy_packages.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (voetberg) [reply docs/operator/policy_packages.md:36]: Sorry yeah that was overly vague. I was envisioning something like: 
```
| - rucio
| - {policy package dir} 
    | - {package} 
        | - __init__.py
        | - permission.py 
        | - schema.py 
        | - {package}.py
```

[the remark this answers is not in the document]
Comment (rdimaio) [reply docs/operator/policy_packages.md:35]: Added in latest revision

**Review by voetberg — APPROVED** [review]

**Pull request merged**